content/docs/security.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Security
+description: Securing TiKV 
+weight: 7
+draft: false
+---
+
+This page discusses how to secure your TiKV deployment. Learn how to:
+
+* Use [Transport Layer Security](#transport-layer-security-tls) to encrypt connections between TiKV nodes
+* Use [On-Disk Encryption](#on-disk-encryption) to encrypt the data that TiKV reads and writes to disk
+* [Report vulnerabilities](#reporting-vulnerabilities)
+
+## Transport Layer Security (TLS)
+
+It is possible to use TLS encryption with TiKV. To start, you need to prepare the certificates for the deployment.
+
+### Prepare Certificates
+
+It is recommended to prepare a separate server certificate for TiKV and PD, and make sure that they can authenticate each other. The clients of TiKV and PD share one client certificate.
+
+You can use multiple tools to generate self-signed certificates, such as `openssl`, `easy-rsa` and `cfssl`.

```

Comment:
```suggestion
You can use multiple tools to generate self-signed certificates, such as `openssl`, `easy-rsa`, and `cfssl`.
```

Reply:
I'm glad we can agree about the oxford comma. :) 

![image](https://user-images.githubusercontent.com/130903/53659555-b9735f00-3c10-11e9-9eff-7b5a9782d621.png)